build-test-saml.xml

```diff
@@ -3,9 +3,9 @@
 <project basedir="." name="portal-test-saml" xmlns:antelope="antlib:ise.antelope.tasks">
 	<import file="build-test.xml" />
 
-	<target name="deploy-keystore-file">
+	<target name="deploy-SAML-keystore-file">
 		<copy
-			file="${project.dir}/portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/dependencies/keystore.jks"
+			file="${project.dir}/portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/dependencies/SAMLKeystore.jks"
```

Comment:
The portal cannot recognize the jks certificate if it is changed to a name other than keystore.jks
See failures:
https://testray.liferay.com/home/-/testray/case_results/2242821270
https://testray.liferay.com/home/-/testray/case_results/2242821201
https://testray.liferay.com/home/-/testray/case_results/2242821221
https://testray.liferay.com/home/-/testray/case_results/2242821429

Reply:
@stian-sigvartsen Is this filename is hardcoded?

Reply:
Solved, I ignored this change.